modules/serverless-ossm-v1x-jwt.adoc

```diff
@@ -6,16 +6,52 @@
 [id="serverless-ossm-v1x-jwt_{context}"]
 = Using JSON Web Token authentication with {SMProductShortName} 1.x and {ServerlessProductName}
 
-You can use the following procedure to enable using JSON Web Token authentication with {SMProductShortName} 1.x and {ServerlessProductName}.
+You can use JSON Web Token (JWT) authentication with Knative services by using {SMProductShortName} 1.x and {ServerlessProductName}. To do this, you must create a policy in the application namespace that is a member of the `ServiceMeshMemberRoll` object. You must also enable sidecar injection for the service.
+
+[IMPORTANT]
+====
+Adding sidecar injection to pods in system namespaces, such as `knative-serving` and `knative-serving-ingress`, is not supported when Kourier is enabled.
+
+ifdef::openshift-enterprise[]
+If you require sidecar injection for pods in these namespaces, see the {ServerlessProductName} documentation on _Integrating {SMProductShortName} with {ServerlessProductName} natively_.
+endif::[]
+====
 
 .Prerequisites
 
-* You have installed the {ServerlessOperatorName} and Knative Serving.
+* You have installed the {ServerlessOperatorName}, Knative Serving, and {SMProductName} installed on your cluster.
 * Install the OpenShift CLI (`oc`).
 * You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
 
 .Procedure
 
+. Add the `sidecar.istio.io/inject="true"` annotation to your service:
++
+.Example service
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: <service_name>
+spec:
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "true" <1>
+        sidecar.istio.io/rewriteAppHTTPProbers: "true" <2>
+...
+----
+<1> Add the `sidecar.istio.io/inject="true"` annotation.
+<2> You must set the annotation `sidecar.istio.io/rewriteAppHTTPProbers: "true"` in your Knative service as {ServerlessProductName} versions 1.14.0 and higher use an HTTP probe as the readiness probe for Knative services by default.
+
+. Apply the `Service` resource:
++
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+
 . Create a policy in a serverless application namespace which is a member in the `ServiceMeshMemberRoll` object, that only allows requests with valid JSON Web Tokens (JWT):
```

Comment:
Nit: In other places we use this form: `a member of the ServiceMeshMemberRoll object` instead of `a member in the ServiceMeshMemberRoll object`. Not sure what is better though.

Reply:
I'll leave this as it is for now since it's not part of the PR at the minute and I want to keep the scope limited.